cv32e40s/tests/cfg/default.yaml

```diff
@@ -3,6 +3,8 @@ description: Default configuration for CV32E40S simulations
 compile_flags:
     +define+ZBA_ZBB_ZBC_ZBS
 ovpsim: >
+    --override cpu/hpmcounter_undefined=T
+    --override root/cpu/envcfg_mask=0x0
```

Comment:
Cant remember to have done this, so not sure if this is supposed to be here or not.

Reply:
Thanks for the updates @silabs-krdosvik, as for your questions:
 hpmcounter_undefined should have been included in the environment, but adding it here is a good reminder to get that fixed. envcfg_mask=0x0 also looks fine, as all the fields are hard wired zero.
 
For the future, please use a descriptive title on PRs (That can be changed on github when you create the PR)- I have no idea what the PR is about when there is just "Bus" in the title. 

Otherwise this PR is good to go on my end!